tests/e2eTest/e2eTest.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { vi } from "vitest";
+import test_suit_0_1_1 from "./v0.1.1";
+import test_suit_0_3_3 from "./v0.3.3";
+import test_suit_0_3_4 from "./v0.3.4";
+
+vi.mock("@airgap/beacon-sdk", () => ({
+  NetworkType: {
+    MAINNET: "mainnet",
+    GHOSTNET: "ghostnet",
+  },
+}));
+
+test_suit_0_1_1();
+test_suit_0_3_3();
+test_suit_0_3_4();
```

Comment:
You way of running the tests works, but it's not really the intended way of doing it. Normally you should:
- Rename `v01.1.1.ts` to `v01.1.1.spec.ts` (and the same for the other files)
- I'm not sure how much it's required, but you have to import `expect`, `it`, `descripe`, from `vitest` ([doc](https://vitest.dev/guide/#writing-tests))

Is there a reason the tests are the way you did it?

Reply:
yes, I want them run in order.
If they are in three different files, they will run in parallel.
It's real time to send ops to ghostnet and I make them use the same key. if you run them in parallel, high chance are those op will fail because they are using the same op counter.
It looks like vitest doesn't support for ordering by passing arg or put annotations.